### DISCUSSION TITLE
DS416Play is Working

### DISCUSSION THREAD

**Discussion by christian-hoekstra** [runfalk/synology-wireguard#70] [Show and tell]
Hi all,

Just want to report that this solution is also compatible with the DS416Play (braswell architecture), which was not listed on the compatibility matrix. Tested using DSM 6.2.3-25426 Update 3. Please note that the DS416 (without the word play has a ARM based architecture and is untested by me). Did not use the pre-compiled package, but compiled it myself by following the instructions provided. As a client I used the IOS WireGuard app. 

Although the solution is not monkey proof, it is a great initiative and a good start! 
So a big thanks from me to all the contributors!

Mvg,

Christian

**Comment by dragonwyc** [comment]
Ds416j is also working by followingthe latest instructions provided, thanks a lot!
